salt/transport/ipc.py

```diff
@@ -20,10 +20,17 @@
 
 import salt.transport.frame
 import salt.utils.msgpack
+from salt.utils.versions import warn_until
 
 log = logging.getLogger(__name__)
 
 
+warn_until(
```

Comment:
see earlier note about dunder deprecate

Reply:
Dunder deprecate only works on things imported by the loader. It won't work here.